spark/dl/src/main/scala/com/intel/analytics/bigdl/models/autoencoder/Train.scala

```diff
@@ -55,6 +55,9 @@ object Train {
 
   def main(args: Array[String]): Unit = {
     trainParser.parse(args, new TrainParams()).map(param => {
+      if (param.debug) {
```

Comment:
why add this?

Reply:
With this statement, users can view debug log by simply add '--DEBUG' in their command line to run bigdl model. It is related to the issue https://github.com/intel-analytics/BigDL/issues/1521. They can view DEBUG log without modifying code now

Reply:
I think you misunderstanding the issue https://github.com/intel-analytics/BigDL/issues/1521.

Reply:
For log4j, you don't need to change log level in this method. You can modify the log level without modifying any code by providing a log property file, and set a JVM property to use that log property file.

Log property file: https://www.mkyong.com/logging/log4j-log4j-properties-examples/
Log4j configuration: https://logging.apache.org/log4j/2.x/manual/configuration.html